src/main/resources/application.properties

```diff
@@ -1,6 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_shop?createDatabaseIfNotExist=true
-spring.datasource.username=root
-spring.datasource.password=root
```

Comment:
This properties still should be present, they just should be empty: spring.datasource.username= or replaced with placeholders or links to system variables spring.datasource.username={DB-USERNAME}

Reply:
Same with env file